src/Models/T1_relaxometry/vfa_t1.m

```diff
@@ -53,7 +53,7 @@
     properties
         MRIinputs = {'VFAData','B1map','Mask'};
         xnames = {'M0','T1'};
-        voxelwise = 1;
+        voxelwise = 0; % Fits are still voxelwise even when this is zero
```

Comment:
We are aware that this name is a bit confusing. 

It's role is to let our framework know that if fit function is going to be called per voxel. This enables/disables some features (such as `View data fit` in the GUI to begin with).